libafl_sugar/src/qemu.rs

```diff
@@ -218,7 +218,7 @@ where
             let mut fuzzer = StdFuzzer::new(scheduler, feedback, objective);
 
             // The wrapped harness function, calling out to the LLVM-style harness
-            let mut harness = |input: &BytesInput| {
+            let mut harness = |input: &BytesInput, _emulator: &mut Emulator<_, _, _, _>| {
```

Comment:
Isn't the emulator global anyway?

Also, if you need it, I feel like Emulator should be at the first position I think.